srcpkgs/linux-asahi/template

```diff
@@ -0,0 +1,349 @@
+# Template file for 'linux-asahi'
+pkgname=linux-asahi
+version=6.12.4p1
+revision=1
+short_desc="Linux kernel and modules for Apple Silicon"
+maintainer="Will Springer <skirmisher@protonmail.com>, dkwo <npiazza@disroot.org>"
+_majver=${version%p*}
+_asahiver=${_majver%.0}
+_asahirel=${version#*p}
+_tag=asahi-${_asahiver}-${_asahirel}
+license="GPL-2.0-only"
+homepage="https://asahilinux.org"
+distfiles="https://github.com/AsahiLinux/linux/archive/${_tag}.tar.gz"
+checksum=6543a6aa3fadb65fa24d5b7d4ae2143054229a94bc309512ce7c436104048e2f
+
+python_version=3
+
+archs="aarch64*"
+
+nodebug=yes  # -dbg package is generated below manually
+nostrip=yes
+noverifyrdeps=yes
+noshlibprovides=yes
+preserve=yes
+
+hostmakedepends="tar xz bc elfutils-devel flex gmp-devel kmod libmpc-devel
+ pkg-config openssl-devel perl uboot-mkimage cpio pahole python3 zstd
+ rust cargo rust-bindgen rust-src libclang"
+
+_kernver=${_majver}-${_tag}_${revision}
+triggers="kernel-hooks"
+kernel_hooks_version="${_kernver}"
+
+# These files could be modified when an external module is built.
+mutable_files="
+ /usr/lib/modules/${_kernver}/modules.builtin.bin
+ /usr/lib/modules/${_kernver}/modules.builtin.alias.bin
+ /usr/lib/modules/${_kernver}/modules.softdep
+ /usr/lib/modules/${_kernver}/modules.dep
+ /usr/lib/modules/${_kernver}/modules.dep.bin
+ /usr/lib/modules/${_kernver}/modules.symbols
+ /usr/lib/modules/${_kernver}/modules.symbols.bin
+ /usr/lib/modules/${_kernver}/modules.alias
+ /usr/lib/modules/${_kernver}/modules.alias.bin
+ /usr/lib/modules/${_kernver}/modules.devname"
+
+# reproducible build
+export KBUILD_BUILD_TIMESTAMP=$(LC_ALL=C date -ud @${SOURCE_DATE_EPOCH:-0})
+export KBUILD_BUILD_USER=voidlinux
+export KBUILD_BUILD_HOST=voidlinux
+
+if [ "$CROSS_BUILD" ]; then
+	_cross="CROSS_COMPILE=${XBPS_CROSS_TRIPLET}-"
+fi
+
+pre_patch() {
```

Comment:
kinda useless if just using git tags (this is for how kernel.org distributes patch releases)

Reply:
Done.